examples/plotting/aspect.py

```diff
@@ -1,3 +1,12 @@
+''' This example demonstrates how a circle with a data-space radius appears
+when plotted with different aspect scales specified.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure, bokeh.plotting.show, bokeh.layouts.layout
+    :refs: :ref:`_ug_basic_layouts`
```

Comment:
It confusing, when *defining* the anchors, Sphinx requires the leading `_` but when *using* the anchors, it should not be present
```suggestion
    :refs: :ref:`ug_basic_layouts`
```

Reply:
Thank you for this. I had noticed the difference but I didn't know if it was a reasonable question to ask. I have updated it now.